docs/openapi/openapi-events-api.yaml

```diff
@@ -1,40 +0,0 @@
-openapi: "3.0.1"
```

Comment:
I removed this spec as it doesn't really provide any value. @alexdebrie thoughts?